Tests/OAuth/ResourceOwner/AzureResourceOwner.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\Tests\OAuth\ResourceOwner;
+
+use HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\AzureResourceOwner;
+
+class AzureResourceOwnerTest extends GenericOAuth2ResourceOwnerTest
+{
+    protected $csrf = true;
+
+    protected $userResponse = <<<json
+{
+    "id":  "1",
+    "foo": "bar"
```

Comment:
@Soullivaneuh Sorry it took so long but I finally got some time to get it fixed properly.

I can't edit the file in this PR, what is the best way to get the code in here?

I'll paste the changed lines in case you are willing to edit them in:
protected $userResponse = <<<json
{
    "sub": "1",
    "given_name": "Dummy",
    "family_name": "Tester",
    "name": "Dummy Tester",
    "unique_name": "dummy123"
}
json;

---- AND
public function testGetUserInformation()
    {
        $token = '.' . base64_encode($this->userResponse);
        /**
         * @var $userResponse \HWI\Bundle\OAuthBundle\OAuth\Response\AbstractUserResponse
         */
        $userResponse = $this->resourceOwner->getUserInformation(array(
            'access_token' => 'token',
            'id_token' =>  $token
        ));

        $this->assertEquals('1', $userResponse->getUsername());
        $this->assertEquals('Dummy Tester', $userResponse->getRealName());
        $this->assertEquals('dummy123', $userResponse->getNickname());
        $this->assertNull($userResponse->getRefreshToken());
        $this->assertNull($userResponse->getExpiresIn());
    }